exampleSite/content/nesting/content.md

```diff
@@ -4,9 +4,7 @@ fragment = "content-single"
 date = "2017-10-05"
 weight = 100
 #background = ""
-description = "Nested Page"
-
-content = """
```

Comment:
Let's add commented and empty titles, just as easy of use for people setting something up:
```
#title = ""
#subtitle = ""
```

Reply:
I removed `# Nested page` and added it as title. Also commented all the subtitles.